frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSyncToParentByRebaseAction.java

```diff
@@ -216,9 +229,8 @@ public void run(ProgressIndicator indicator) {
         if (maybeExecutionResult != null && maybeExecutionResult.getExitCode() != 0) {
           val message = "machete-pre-rebase hooks refused to rebase (exit code ${maybeExecutionResult.getExitCode()})";
           LOG.error(message);
-          val executionResult = maybeExecutionResult;
```

Comment:
Actually... it looks it can be just called `executionResult` now (isn't wrapped in any `Option` or `Try`)